### PR TITLE
MAINT: Drop manylinux1 and manylinux2010 wheels in main.

### PR DESCRIPTION
**Pull request by charris**
Start using manylinux2014 for nightly builds in preparation for NumPy 1.22.x.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -49,60 +49,44 @@ jobs:
       name: linux
       vmImage: ubuntu-18.04
       matrix:
+        # manylinux 2014
+        pypy_3.7_64:
+          MB_PYTHON_VERSION: "pypy3.7-7.3.4rc1"
```
Comment (mattip) [inline azure-pipelines.yml:54]: PyPy 7.3.5 was released.
```suggestion
          MB_PYTHON_VERSION: "pypy3.7-7.3.5"
```

**Comment by charris** [comment]
Updated. I was going to update pypy in another PR, but went ahead and did it here. Next up, ILP64. What platforms is that available on?

**Comment by charris** [comment]
ARM64 is still acting up.

**Comment by mattip** [comment]
> Next up, ILP64.

Any that have ` openblas64` in the name at https://anaconda.org/multibuild-wheels-staging/openblas-libs/files, which should be all of them. Good thing you asked, it seems the [ppc64 build failed](https://travis-ci.com/github/MacPython/openblas-libs/builds/231475410) on the merge-to-master and I didn't notice. I retriggered it.

**Comment by charris** [comment]
Hmm, looks like the easiest way to pass `NPY_USE_BLAS_ILP64` into docker is to use a new `env_vars_ilp64.sh` file similar to `env_var_32.sh`.

**Comment by mattip** [comment]
xref MacPython/openblas-libs#59

**Pull request merged**